web3/module.py

```diff
@@ -65,8 +68,11 @@ def retrieve_async_method_call_fn(
 ) -> Callable[..., Coroutine[Any, Any, RPCResponse]]:
     async def caller(*args: Any, **kwargs: Any) -> RPCResponse:
         (method_str, params), response_formatters = method.process_params(module, *args, **kwargs)
-        result_formatters, error_formatters = response_formatters
-        result = await w3.manager.coro_request(method_str, params, error_formatters)
+        result_formatters, error_formatters, null_response_formatters = response_formatters
```

Comment:
@kclowes [small but clarifying change] Rename suggestion from `null_response_formatters` here to `null_result_formatters` just to keep the naming consistent across files.